template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hdfs/HdfsVolumeConfigProvider.java

```diff
@@ -40,7 +39,7 @@
             case HdfsRoles.JOURNALNODE:
                 return List.of(
                         config("dfs_journalnode_edits_dir",
-                                buildSingleVolumePath(volumeCount, "journalnode"))
+                                buildVolumePathStringZeroVolumeHandled(volumeCount, "journalnode"))
```

Comment:
Are you sure it accepts multiple volumes?

Reply:
I think this is the HDFS code where the property is used, seems to confirm your findings:

https://github.com/apache/hadoop/blob/c4ba0ab7dfcfe76f928d06fcb29ca0cc4d778737/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournalNode.java#L178-L182
https://github.com/apache/hadoop/blob/c4ba0ab7dfcfe76f928d06fcb29ca0cc4d778737/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournalNode.java#L335-L340

Reply:
Yea based on the code @adoroszlai shared and also the docs in hdfs-default.xml:

```
<property>
  <name>dfs.journalnode.edits.dir</name>
  <value>/tmp/hadoop/dfs/journalnode/</value>
  <description>
    The directory where the journal edit files are stored.
  </description>
</property>
```

The journal edits dir can only have a single value. This makes it different from the NN data directory where the namenode can have a comma separated list of directories.

Reply:
And fixed.